assets/css/styles.css

```diff
@@ -0,0 +1,525 @@
+main {
+  margin-top: 5em;
+  background-color: #f4f5f7;
+}
+
+main h3 {
+  font-size: 16px;
+  color: #a0a0a0;
+  font-weight: bold;
+}
+
+main p {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#subs-btn {
+  font-family: "Roboto Condensed", sans-serif;
+  font-size: 10px;
+  background-color: yellow;
+  border-radius: 60px;
+}
+
+body main div div section section div h2,
+body main div div section section div h2 a {
+  color: black;
+  font-size: 22px;
+  margin-top: -1em;
+}
+
+.debate-text a {
+  color: black;
+}
+
+.itmag-cards h2,
+.itmag-cards a {
+  font-family: "Roboto Condensed", sans-serif;
+  font-size: 1.3em;
+  line-height: 1em;
+  color: black;
+  text-decoration: none;
+}
+
+.aside-news a {
+  font: bold 17px/21px "Roboto Condensed", sans-serif;
+  color: black;
+  text-decoration: none;
+}
+
+.header-lg a {
+  color: black;
+  padding: 0 auto;
+  font-weight: bold;
+  font-size: 18px;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+main a:hover {
+  text-decoration: none;
+}
+
+.navbar {
+  background-color: red;
+}
+
+footer div img {
+  width: 236px;
+}
+
+.navbar-brand img {
+  height: 60px;
+}
+
+.navbar-brand-2 img {
+  height: 20px;
+}
+
+.cards {
+  min-width: 290px;
+}
+
+.cards img {
+  width: 100%;
+  height: auto;
+}
+
+.h-title-sponsor {
+  color: black;
+  font-size: 18px;
+}
+
+.sponsor-txt h2 a {
+  font-size: 20px;
+}
+
+.heroes-txt h2 {
+  font-size: 15px;
+  color: #333;
+}
+
+.heroes-txt h2 a {
+  font-size: 15px;
+  color: #333;
+}
```

Comment:
Don't repeat yourself, in order to minimize code you can group selectors with the same style definition